### PR TITLE
operating order on corp cards

### PR DESCRIPTION
**Pull request by yzemaze**
second part of fix for #1122

### CODE REVIEW

File: assets/app/view/game/corporation.rb [hunk 1]
```diff
@@ -52,8 +52,17 @@ def render
...
-          children << h('table.center', [h(:tr, subchildren)])
+          props = {
+            style: {
+              grid: '1fr / repeat(2, max-content)',
+              gap: '2rem',
+              justifyContent: 'center',
+            },
+          }
+
+          subchildren = (n = @game.round.entities.find_index(@corporation)) ? [render_operating_order(n)] : []
```
Comment (tobymao) [inline assets/app/view/game/corporation.rb:63]: %s/find_index/index/g

Comment (tobymao) [reply assets/app/view/game/corporation.rb:63]: it'd be cleaner if it was just 

sub_children = render_operating_order(@game.round.entities.index(@corporation))

File: assets/app/view/game/corporation.rb [hunk 2]
```diff
@@ -303,7 +312,13 @@ def render_shares
 
       def render_revenue_history
         last_run = @corporation.operating_history[@corporation.operating_history.keys.max].revenue
-        h('td.bold', "Last Run: #{@game.format_currency(last_run)}")
+        h('div.bold', "Last Run: #{@game.format_currency(last_run)}")
+      end
+
+      def render_operating_order(n)
+        round = @game.round
```
Comment (tobymao) [inline assets/app/view/game/corporation.rb:319]: don't need to store this var you only use it once

File: assets/app/view/game/corporation.rb (hunk 1 above)
Comment (yzemaze) [reply assets/app/view/game/corporation.rb:63]: yep, but won’t work for one test case:

```
Assets #html 1889 renders float
     Failure/Error: context.eval(script, filename: @file)
     
     MiniRacer::RuntimeError:
       >=: undefined method `>=' for nil
```

hence: assignment ? …

File: assets/app/view/game/corporation.rb (hunk 2 above)
Comment (yzemaze) [reply assets/app/view/game/corporation.rb:319]: twice (was three times, hence it’s there). Where’s your cutoff? ;)

Comment (tobymao) [reply assets/app/view/game/corporation.rb:319]: ah didn't see you used it twice that's fine

File: assets/app/view/game/corporation.rb (hunk 1 above)
Comment (tobymao) [reply assets/app/view/game/corporation.rb:63]: i mean obviously if you copy what i did literally, but you can write code to make it work, like using a compact or returning an empty array if nil in your function

Comment (yzemaze) [reply assets/app/view/game/corporation.rb:63]: sure, had that, but this looked cleaner to me …
I’ll change it again.

**Pull request merged**